src/main/java/net/neoforged/neoforge/data/loading/DatagenModLoader.java

```diff
@@ -35,10 +35,35 @@ public static boolean isRunningDataGen() {
     }
 
     @ApiStatus.Internal
-    public static void begin(final Set<String> mods, final Path path, final Collection<Path> inputs, Collection<Path> existingPacks,
-            final boolean devToolGenerators, final boolean reportsGenerator,
-            final boolean structureValidator, final boolean flat, @Nullable final String assetIndex, @Nullable final File assetsDir, Runnable setup, GatherDataEvent.GatherDataEventGenerator eventGenerator,
+    public static void begin(
+            final Set<String> mods,
+            final Path path,
+            final Collection<Path> inputs,
+            Collection<Path> existingPacks,
+            final boolean devToolGenerators,
+            final boolean reportsGenerator,
+            final boolean structureValidator,
+            final boolean flat,
+            Runnable setup,
+            GatherDataEvent.GatherDataEventGenerator eventGenerator,
             DataGenerator vanillaGenerator) {
+        begin(mods, path, inputs, existingPacks, devToolGenerators, reportsGenerator, structureValidator, flat, setup, eventGenerator, vanillaGenerator, c -> {});
```

Comment:
Nit: Any reason that this can't be inlined into the call sites, so that the bouncer method isn't needed?